### PR TITLE
Phase 3 Funding: Open call | Add fits maintainer FR

### PR DESCRIPTION
**Pull request by eteq**
Just what it says on the tin.  Note I have not yet had a chance to reach out to @saimn on this, so it's not very fleshed out. But I very much want any input you might have, @saimn (including anything from "yes these are the 10 things that need doing" to "nah, don't do it"!)

### CODE REVIEW

**Comment by hamogu** [comment]
There is > 150 open issues on `io.fits` and, as the proposal says this is indeed one of the most used parts of astropy. Maybe we can pick 5 high-priority items from that list and then say "select and work on (and ideally solve) 30 others"? Given the importance of fits, I would support this to be a much larger funding request (I know we have a lot of requests this year), > 50k$ (i.e. significant full-time work). This is a complex package and I'm afraid that 10 k$ is far too little to get a new person into fits. By the time they understand the package, they've run out of money. If we want to get someone without prior fits experience (and likely without a deep knowledge of the fits conventions and associated papers), then this funding must last long enough that they can learn all this stuff and then have time left to work on issues productively.

**Comment by eteq** [comment]
Alright I tried to address @hamogu's point while staying conscious of some feedback from @saimn on this that this proposal could yield more work for him and therefore do no good at relieving the burden. Given our funding rule that funding should first support volunteers, I think it's critical to respect that. But it is at fundamental tension with @hamogu's point that we want to make sure we build expertise in this very complex area. So the proposal is now explicit about an amount (on the high end of the rang from before) and that this is a *starting* point on growing expertise, limited mainly by not wanting to overwhelm the maintainer base (especially @saimn)

**Comment by aaryapatil** [comment]
Please react to this comment to vote on this proposal (👍, 👎, or no reaction for +0)

**Comment by pllim** [comment]
Haven't heard from @saimn on this. What does he think?

**Comment by saimn** [comment]
We discussed that offline with Erik so the current version of the proposal is fine by me :)

**Comment by pllim** [comment]
Oh no... was I supposed to be reacting to a certain comment? I have been reacting to the original posts.

**Comment by aaryapatil** [comment]
@pllim Ideally, you should react to this comment: https://github.com/astropy/astropy-project/pull/280#issuecomment-1222521970. This way we can keep track of who is voting rather than just reacting in general.

File: finance/proposal-calls/cycle3/fits-io-FR.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Note an open hire of this sort would be expected to continue in another year if all goes well. This proposal would this be a *start* at growing relevant expertise, and it might be multiple cycles before a fresh community member could become a co-maintainer
+
+
+### Approximate Budget
+Exact scope would be kept flexible (see emphasis above of not overwhelming maintainers). At a fiducial hourly rate of ~$120/hr and an assumption of ~2 issues per month over a year (assuming about half of the issues could be reviewed by a generalist and half by a specialist, keeping it to one review a month for @saimn at max), and assuming 4hr per issue, a starting estimate might be $11520
+
+#### Minimum budget
+
+In principle the true minimum is a *single* issue - ~$480k. But in practice the minimum.is whatever recoups the time spent on the seaexy as part of the open call.
```
Comment (astrofrog) [inline finance/proposal-calls/cycle3/fits-io-FR.md:20]: $480,000 per issue? Where do I sign up!! 😉

Comment (astrofrog) [reply finance/proposal-calls/cycle3/fits-io-FR.md:20]: Oh wait this is io.fits

Comment (pllim) [reply finance/proposal-calls/cycle3/fits-io-FR.md:20]: srsly though, is this a typo?

Comment (aaryapatil) [reply finance/proposal-calls/cycle3/fits-io-FR.md:20]: I think he meant 480 for a single issue (120$/hr for 4 hrs per issue).

**Comment by kelle** [comment]
Congratulations!  The community has expressed support for this FR, and the Finance Committee has determined it can be funded.  This FR has been approved for $11,520, to be funded from the NASA grant.

@aaryapatil is the Finance Committee member who will be supporting this project.  They will update this PR with the amount and then merge it, as well as being in contact with you to set up any necessary agreements and relevant Open Collective financial arrangements. They will also open a new issue where progress on this project can be discussed.

**Pull request merged**